nevergrad/functions/multiobjective/tests/test_structures.py

```diff
@@ -0,0 +1,98 @@
+from unittest import TestCase
```

Comment:
In the rest of the code, we have put test files alongside the actual files. I find it often more convenient, although that can become messy when there are a lot of files. Since there are not many files in the multiobjective folder, you can probably put the tests directly in in as well.

Reply:
I will replace the `unittest` tests with `pytest`, thank you. I used the tests there for development purposes, and it just was more convenient to have `unittest` at that time. I totally agree these should be `pytest` in the end. Thank you!